README.md

```diff
@@ -4,7 +4,12 @@ Command line tools for [Cloudflare](https://cloudflare.com)
 
 ## Installation
 
-To install, place `cloudflare-cli` anywhere in your PATH. Optionally alias it to something easier, like `cf`.
+To install, place `cloudflare-cli` anywhere in your PATH. Optionally alias it to something easier, like `cfcli`.
+Or just run the following command to install it as `cfcli`.
+
+```bash
+sudo wget -qO cfcli https://git.io/cfcli && sudo chmod +x ./cfcli && sudo install ./cfcli /usr/local/bin/cfcli
```

Comment:
`sudo` sometimes help for some users. `/usr/local/bin` means no need to worry about having it in the $PATH since it already is. `git.io` is from the GitHub family, I don't think there is anything bad with it. It helps keep the command small. Everyone does that?

Let me know what you think now?